### ISSUE TITLE
no instance of NFData1 Vector

### ISSUE THREAD

**Issue by ghorn** [haskell/vector#216]
I think it might look something like:
```haskell
instance NFData1 V.Vector where
  liftRnf arnf = V.foldl' (\_ x -> arnf x) ()
  {-# INLINEABLE liftRnf #-}
```
Maybe you want to define `NFData` in terms of `NFData1`

**Comment by RyanGlScott** [comment]
This sounds reasonable to me. Care to submit a patch?

**Comment by peddie** [comment]
See https://github.com/haskell/vector/pull/233 for a patch.

**Comment by RyanGlScott** [comment]
Fixed in #233.
